tests/test_sdr.py

```diff
@@ -141,11 +141,11 @@ def test_base_regression(self):
 
         sdr.execute(args)
         expected_results = {
-            'usle_tot': 13.90210914612,
-            'sed_export': 0.55185163021,
-            'sed_dep': 8.80130577087,
-            'avoid_exp': 57971.87890625,
-            'avoid_eros': 1458232.5,
+            'usle_tot': 2.62457418442,

```

Comment:
These are some dramatic changes! I know we didn't have a unit test for `_calculate_ls_factor` before, but might I suggest adding one? It would be great to confirm that the new results are as expected on some fake data where we can easily spot-check the math.